packages/components/src/Form/Inputs/Checkbox/Checkbox.spec.tsx

```diff
@@ -49,10 +50,45 @@ const runTimers = () =>
   })
 
 describe('Checkbox', () => {
+  test('renders', () => {
+    renderWithTheme(<Basic />)
+    expect(screen.getByRole('checkbox')).toBeInTheDocument()
+  })
+
+  test('renders checked', () => {
+    renderWithTheme(<Checked />)
+    const checkboxInput = screen.getByRole('checkbox')
+    expect(checkboxInput).toBeInTheDocument()
+    expect(checkboxInput as HTMLInputElement).toBeChecked()
```

Comment:
`as HTMLInputElement` shouldn't be needed - poking at it locally tests appear to work without explicitly forcing the typing on these. Can you remove this explicit assignments?

```suggestion
    expect(checkboxInput).toBeChecked()
```